app/scripts/models/account.js

```diff
@@ -110,7 +113,7 @@ define([
     isEmpty: function () {
       var self = this;
       return ! _.find(ALLOWED_KEYS, function (key) {
-        return typeof self.get(key) !== 'undefined';
+        return self.get(key) !== DEFAULTS[key];
```

Comment:
The semantics now match what the original intention was: is this a default account object? It just so happened all the defaults were undefined previously. I can rename it to `isDefault` for clarity.